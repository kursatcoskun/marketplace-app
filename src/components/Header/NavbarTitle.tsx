import styled from "styled-components";

const NavbarTitle = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  margin-left: auto;
  margin-right: auto;
  font-size: 150%;
  padding: 12px 16px;
`;

export default NavbarTitle;
