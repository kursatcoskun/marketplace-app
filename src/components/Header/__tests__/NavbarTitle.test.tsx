// MyComponent.test.js
import { shallow } from "enzyme";
import NavbarTitle from "../NavbarTitle";
describe("NavbarTitleComponent", () => {
  it("should render NavbarItem component", () => {
    const wrapper = shallow(<NavbarTitle />);
  });

  it("should render initial layout", () => {
    const component = shallow(<NavbarTitle />);
    expect(component.getElements()).toMatchSnapshot();
  });
});
