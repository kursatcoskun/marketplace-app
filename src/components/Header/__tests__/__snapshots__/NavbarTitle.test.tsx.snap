// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarTitleComponent should render initial layout 1`] = `
Array [
  <div
    className="sc-bdnxRM eaiRLH"
  />,
]
`;
